src/Tickets/Emails/Admin/Preview_Modal.php

```diff
@@ -240,42 +240,32 @@ public function get_modal_content_ajax( $render_response, $vars ) {
 			$preview_context['add_qr_codes'] = tribe_is_truthy( $add_qr_codes );
 		}
 
-		$preview_context['add_event_links'] = tribe_is_truthy( Arr::get( $vars, 'eventLinks', '' ) );
-		$preview_context['add_ar_fields']   = tribe_is_truthy( Arr::get( $vars, 'arFields', '' ) );
+		$preview_context['add_event_links']    = tribe_is_truthy( Arr::get( $vars, 'eventLinks', '' ) );
+		$preview_context['add_ar_fields']      = tribe_is_truthy( Arr::get( $vars, 'arFields', '' ) );
+		$preview_context['using_ticket_email'] = tribe_is_truthy( Arr::get( $vars, 'useTicketEmail', '' ) );
 
 		$current_email = Arr::get( $vars, 'currentEmail', '' );
 
-		$html = '';
+		// Show Ticket email by default.
+		$email_class = tribe( Ticket::class );
 
-		if ( ! empty( $current_email ) ) {
+		// Select email class to preview, if not using ticket email.
+		if ( ! $preview_context['using_ticket_email'] && ! empty( $current_email ) ) {
```

Comment:
Will `$preview_context['using_ticket_email']` exist for all emails? If not, should we add an `isset()` check?

Reply:
I think that'd be covered with the `tribe_is_truthy` when getting the context variables [here](https://github.com/the-events-calendar/event-tickets/pull/2651/files#diff-af84e6a443a1d268dc39223a85113a294b3f98c03da30c74970b1c1324b0546cR245).